test-bdd/main_test.go

```diff
@@ -0,0 +1,98 @@
+package test
```

Comment:
Can you make sure we run this test when we do a "make test" from upgrade-manager repo?
If the make target already runs this, ignore this comment.